Classes/Importer/AbstractImporter.php

```diff
@@ -551,27 +601,46 @@ protected function getLabelFromRecordData(array $data)
      *
      * The storage node is either created or just retrieved and finally stored in $this->storageNode.
      *
-     * @param string $nodePath Absolute or relative (to the site node) node path of the storage node
+     * @param string $nodePathOrIdentifier Absolute or relative (to the site node) node path of the storage node
      * @param string $title Title for the storage node document
      * @return void
      * @throws \Neos\ContentRepository\Exception\NodeTypeNotFoundException
      */
-    protected function initializeStorageNode($nodePath, $title)
+    protected function initializeStorageNode($nodePathOrIdentifier, $title)
+    {
+        if (is_string($nodePathOrIdentifier) && $nodePathOrIdentifier[0] === '#') {
+            $this->storageNode = $this->rootNode->getContext()->getNodeByIdentifier(\substr($nodePathOrIdentifier, 1));
+        } else {
+            preg_match('|([a-z0-9\-]+/)*([a-z0-9\-]+)$|', $nodePathOrIdentifier, $matches);
+            $nodeName = $matches[2];
+            $uriPathSegment = Slug::create($title)->getValue();
+
+            $storageNodeTemplate = new NodeTemplate();
+            $storageNodeTemplate->setNodeType($this->nodeTypeManager->getNodeType($this->storageNodeTypeName));
+
+            $this->storageNode = $this->getSiteNode()->getNode($nodePathOrIdentifier);
```

Comment:
This line could be moved further up while all the title building and such should go into the condition #622